include/libtrading/proto/fix_message.h

```diff
@@ -188,6 +188,9 @@ struct fix_message {
 	struct fix_field		*fields;
 };
 
+#define FIX_PARSE_CONVENTIONAL	0x0
+#define FIX_PARSE_FAST		0x1
```

Comment:
Why not make this a bitmask instead? Have something like:

```c
enum {
    FIX_PARSE_NO_CSUM = 1UL << 0,
};
```

and pass that to ``fix_message_parse()`` (remember to make ``flags`` unsigned).